mlflow/sklearn/__init__.py

```diff
@@ -152,6 +152,7 @@ def save_model(
     input_example: ModelInputExample = None,
     pip_requirements=None,
     extra_pip_requirements=None,
+    pyfunc_predict_func="predict",
```

Comment:
Nit: Can we use `fn` instead of `func`?

```suggestion
    pyfunc_predict_fn="predict",
```